src/components/PaginatedTable/PaginatedTable.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/theme/index.css';
+:root {
+  --table-footer-background-color: var(--structure-color-200);
+  --table-font-family: 'Inter';
+  --table-header-color: var(--structure-color-900);
+  --table-font-family: 'Inter';
+  --table-footer-font-weight: 300;
+  --table-padding: 10px;
+  --table-font-size: 14px;
+}
+
+tfoot td.paginated-table-action-btns {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  overflow: hidden;
+}
+
+.paginated-table-action-btns .next-btn-table-paginated,
+.paginated-table-action-btns .prev-btn-table-paginated {
+  font-weight: var(--table-header-font-weight);
```

Comment:
@branquinhoaa please make sure all the CSS property definitions are ordered alphabetically. This is to make sure that we facilitate debugging and locating the properties.